### ISSUE TITLE
Verbose = 0 still display widget

### ISSUE THREAD

**Issue by shllgtca** [keras-team/keras#10732]
hi everyone, I'm new around, but I have an easy issue: On the code below, I defined verbose =0 but the widget still been displayed, as you can see on the msg showed in the image.

#CODE:
for i in range(NUM_EPOCHS):
    print("Epoch {:d}/{:d}".format(i+1, NUM_EPOCHS))
    model.fit(Xtrain, Ytrain, batch_size=BATCH_SIZE, epochs=1, verbose=0,
              validation_data=(Xtest, Ytest),
              shuffle=False, callbacks=[TQDMNotebookCallback()])
    model.reset_states()

![captura de tela 2018-07-19 as 4 30 11 pm](https://user-images.githubusercontent.com/41371818/42965760-35bdabe8-8b71-11e8-9f63-dfd0ef157f2d.png)

It sounds that the frontend does not support the widget, but I thought that once the verbose =0 it would not have any attempt to display the widget!

**Comment by Dref360** [comment]
Could this be an issue with TQDMNotebookCallback()? Please contact the maintainer of this Callback. It's not related to Keras.

**Comment by shllgtca** [comment]
Thanks, it was exactly it. solved.
